mail/src/main/java/com/sun/mail/util/logging/CompactFormatter.java

```diff
@@ -474,12 +484,7 @@ private String findAndFormat(final StackTraceElement[] trace) {
      */
     private String formatStackTraceElement(final StackTraceElement s) {
         String v = simpleClassName(s.getClassName());
-        String result;
-        if (v != null) {
-            result = s.toString().replace(s.getClassName(), v);
-        } else {
-            result = s.toString();

```

Comment:
FYI this is dead code caught by code coverage.  It seems from the API docs that StackTraceElement::getClassName is non null.  Other parts of the code already treat it as non-null.